### DISCUSSION TITLE
Is there a way to export profiles in the android app?

### DISCUSSION THREAD

**Discussion by Jafaral** [strongswan/strongswan#1705] [Q&A] [answered]
The Android app can import profiles, is there a way to export profiles too?

**Comment by tobiasbrunner** [comment, marked as answer]
No, currently not. But they can be created quite easily manually, see [the docs](https://docs.strongswan.org/docs/5.9/os/androidVpnClientProfiles.html).

**Reply by betapl3b** [reply]
Hello. Is there a way to get password from profile of android strongswan client?
I've lost my password but want to configure same profile on the new phone

**Reply by tobiasbrunner** [reply]
No, currently not (at least nothing built-in, it might be possible to access the database via `adb`).
